### ISSUE TITLE
Accessibility: Low text contrast for toast notifications

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#5026]
### Steps To Reproduce

1. Trigger a success or failure notification

### Expected Result

Foreground/background colours used for the notification should have a contrast ratio of at least 4.5:1

### Actual Result

Most toasts (depending on theme chosen) have insufficient text colour contrast. This issue spans web, desktop, and browser extension.

### Screenshots or Videos

A selection of error toasts, and one success toast. I did not grab all possible screenshots, but suggest most toasts (success, error, and any other types if they're used anywhere) should be reviewed.

Desktop app - theme: dark. background colour: `#FF3E24` / contrast ratio: 3.52:1
![dark-toast-error](https://user-images.githubusercontent.com/895831/226210653-c642d1b1-2945-41cf-81d4-c8d43d7fb006.png)

Desktop app - theme: light. background colour: `#DD4B39` / contrast ratio: 4.09:1
![light-toast-error](https://user-images.githubusercontent.com/895831/226210654-b6742556-58f0-4d8f-a136-b42a57b7e7bb.png)

Desktop app - theme: nord. background colour: `#BF616A` / contrast ratio: 4.09:1
![nord-toast-error](https://user-images.githubusercontent.com/895831/226210656-49cae4ea-b3c8-45e2-b1a5-f1f600f12307.png)

Desktop app - them: nord. background colour: `#A3BE8C` / contrast ratio: 2.04:1
![nord-toast-success](https://user-images.githubusercontent.com/895831/226210657-86f162d5-bf01-4a35-94ea-6b78e9f1eb2e.png)

The only good contrast one I spotted:

Browser extension - theme: solarized dark. background colour: `#DC322F` / contrast ratio: 4.63:1
![solarized-dark-toast-error](https://user-images.githubusercontent.com/895831/226210658-88741e4c-af1c-4cc7-82e9-64e9c886ef1b.png)

### Operating System

Windows

### Web Browser

Chrome

### Build Version

2023.2.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Greenderella** [comment]
Hi there,

Thank you for your report! I have flagged this to our engineering team.

If you or anybody else wishes to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by patrickhlauke** [comment]
Possibly worth looping in the design team /cc @danielleflinn

**Comment by Hinton** [comment]
Hi @patrickhlauke,

Toasts are on our long list of things to do, we have a new design https://www.figma.com/file/Zt3YSeb6E6lebAffrNLa0h/Tailwind-Component-Library?node-id=1717-14573&t=Qronbe3t4gfWkvmU-4 but it will take a while for it to trickle down to all the clients.

**Comment by patrickhlauke** [comment]
Thanks for the background @Hinton - confirming that the colour combinations I'm seeing in that Figma design all pass contrast requirements. Suggest keeping this issue open until the new designs roll out? Or closing this now as it's in progress?

**Comment by danielleflinn** [comment]
@patrickhlauke Yes the text / background color combinations in Figma have been checked for WCAG AA compliant contrast. We are still working toward a better theming strategy; it will likely be similar to what you described with 2 official Bitwarden WCAG compliant themes, and potentially offer users the ability to upload their own theme.

I'm in favor of keeping this issue open until the contrast issues are fixed.

**Comment by patrickhlauke** [comment]
> Yes the text / background color combinations in Figma have been checked for WCAG AA compliant contrast.

@danielleflinn sorry, didn't mean to imply that you didn't check them yourselves btw. was just stating the obvious here for completeness :)
